_posts/2018-11-06-simple-slackbot.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title:  "Simple Slackbot"
+date: 2018-11-06 00:03:30 +0200
+categories: javascript
+author_name: Sergii Paryzhskyi
+author_url : /author/sergii_paryzhskyi
+author_avatar: sergii_paryzhskyi
+read_time : 7
+show_related_posts: false
+square_related: recommend-wolf
+feature_image: posts/2018-11-06-simple-slackbot/poster.jpg
+---
+
+Recently I've built a small slackbot that is posting with certain regularity in specified slack-channel. I will tell you about the idea behind it and also share some technical details.
```

Comment:
slack-channel -> Slack channel. I would also go with capital S for Slack as a product name.